app/models/base.py

```diff
@@ -1,7 +1,29 @@
-from sqlalchemy.orm import DeclarativeBase
+# `func`을 추가하여 SQL 기본 함수 활용
+from sqlalchemy import DateTime, func
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+from uuid import UUID, uuid4
+from datetime import datetime
+import pytz
 
+# 한국 시간대 설정
+kst = pytz.timezone("Asia/Seoul")
 
-# Base 클래스는 SQLAlchemy의 모든 ORM 모델 클래스가 상속받는 기본 클래스입니다.
-# 이 클래스는 데이터베이스 테이블과 매핑되는 ORM 모델을 정의하는 데 사용됩니다.
+
+# 공통된 칼럼을 상속받는 Base 클래스
 class Base(DeclarativeBase):
-    pass  # 이 클래스 자체는 비어 있지만, 상속받는 클래스들이 테이블을 정의하게 됩니다.
+    """Base class which provides automated table name
+    and surrogate primary key column.
+    """
+
+    id: Mapped[UUID] = mapped_column(
+        UUID(as_uuid=True), primary_key=True, default=uuid4
+    )
+
+    created_at: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True),
+        server_default=func.now(),
```

Comment:
한줄로 있어도 되는 것 같습니다. `updated_at`이 더 길지만 분리가 안되어 있는걸 보면 이거도 한줄에 써도 될듯합니당.

```suggestion
        DateTime(timezone=True), server_default=func.now(),
```